packages/ui-default/entry.js

```diff
@@ -46,10 +46,11 @@ document.addEventListener('DOMContentLoaded', async () => {
   // eslint-disable-next-line camelcase
   try { __webpack_public_path__ = UiContext.cdn_prefix; } catch (e) { }
 
-  const [data] = await Promise.all([
-    (await fetch(`/constant/${UiContext.constantVersion}`, { cache: 'force-cache' })).json(),
-    await import('./modules'),
+  const [data, HydroExports] = await Promise.all([
```

Comment:
api includes utils, loading will error because getLang load before LANGS load.
```suggestion
  window.LANGS = {};
  const [data, HydroExports] = await Promise.all([
```